### PR TITLE
#161966619  user should be able to see the the list and profile of other users

### PR DESCRIPTION
**Pull request by salma-nyagaka**
### What does this pull request do?
 Gets a list of all existing authors profile 
### Description of the tasks to be completed?
- add functionality to enable authenticated users to get a list of all existing author profile
### How should this be manually tested?
- clone the repo `git clone https://github.com/andela/ah-technocrats.git`
- `cd ah-technocrats`
- checkout out to this branch `git checkout ft-list-users-161966619 `
- make virtual environment `virtualenv venv`
- activate the virtual environment `source venv/bin/activate`
- install all the dependencies from the requirements file ` pip install -r requirements.txt`
- make migrations  `python manage.py makemigrations`
- apply the migrations `python manage.py migrate`
- run the server  `python manage.py runserver`
- go to Postman to confirm the functionality.
### Any background context you want to provide?
- Once a user logs in, they can access be able to get the list and profiles of existing authors.
### What are the relevant Pivotal Tracker Stories
[#161966619](https://www.pivotaltracker.com/story/show/161966609)
# Checklist:
- [x] My code follows the style guidelines of this project
- [ ] At least 2 people have reviewed my PR
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] My PR has one commit.

screen shot 2018-12-07 at 10 33 40

### CODE REVIEW

**Comment by Mnickii** [comment]
Rebase

File: authors/apps/profiles/urls.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from django.urls import path
-from .views import RetrieveProfileAPIView
+from .views import RetrieveProfileAPIView, RetrieveAuthorsAPIView
 
 urlpatterns = [
-    path('profiles/<username>/', RetrieveProfileAPIView.as_view(), name='profile')
+    path('profiles/<username>/', RetrieveProfileAPIView.as_view(), name='profile'),
+    path('authors/', RetrieveAuthorsAPIView.as_view(), name='authors')
```
Comment (Mnickii) [inline authors/apps/profiles/urls.py]: let's maintain consistency here. Are we going to stick with users or use authors?

Comment (Mnickii) [reply authors/apps/profiles/urls.py]: Choose either to refer to our users as users or authors and let it be reflected in the endpoints. We already have urls names users. Are users separate from authors? If not, let's maintain one.

**Review by Mnickii — CHANGES_REQUESTED** [review]

File: authors/apps/profiles/urls.py (hunk 1 above)
Comment (salma-nyagaka) [reply authors/apps/profiles/urls.py]: Do you mean I should modify the name of the url path? 
They are not separate from authors

**Comment by Mnickii** [comment]
rebase

**Review by Mnickii — APPROVED** [review]

**Pull request merged**